build.gradle

```diff
@@ -36,8 +36,8 @@ subprojects {
     ext['mockito.version'] = '2.28.2'
     ext['hamcrest-library.version'] = '1.3'
     ext['slf4j-log4j.version'] = '1.7.25'
-    ext['reactor-netty.version'] = '0.7.15.RELEASE'
     ext['okhttp.version'] = '3.14.2'
```

Comment:
We should bump soon to 4.0.2, which is kotlin *source* incompatible.

Reply:
Should do that on next versions bump round